modules/cluster-logging-release-notes-5.1.0.adoc

```diff
@@ -0,0 +1,68 @@
+[id="cluster-logging-release-notes-5-1-0"]
+= OpenShift Logging 5.1.0
+
+This release includes link:https://access.redhat.com/errata/RHSA-2021:2112[RHSA-2021:2112 OpenShift Logging Bug Fix Release 5.1.0].
+
+[id="openshift-logging-5-1-0-new-features-and-enhancements"]
+== New features and enhancements
+
+OpenShift Logging 5.1 now supports {product-title} 4.7 and later running on:
+
+* IBM Power Systems
+* IBM Z and LinuxONE
+
+This release adds improvements related to the following components and concepts.
+
+* As a cluster administrator, you can use Kubernetes pod labels to gather log data from an application and send it to a specific log store. You can gather log data by configuring the `inputs[].name.application.selector.matchLabels` element in the `ClusterLogForwarder` custom resource (CR) YAML file. You can also filter the gathered log data by namespace.
```

Comment:
It should be inputs[].application.selector.matchLabels

Reply:
An example:
  inputs:
    - name: myLogsQA1
      application:
        selector:
          matchLabels:
            run: centos-logtest-qa-1
        namespaces: project-qa-1

Reply:
Done.

Reply:
Replaced with `inputs[].application.selector.matchLabels`.